src/connector/Cargo.toml

```diff
@@ -34,6 +34,7 @@ aws-sdk-s3 = { workspace = true }
 aws-smithy-http = { workspace = true }
 aws-smithy-runtime-api = { workspace = true }
 aws-smithy-types = { workspace = true }
+aws-smithy-types-convert = { version = "0.60.1", features = ["convert-chrono"] }
```

Comment:
Better to add this to workspace dependency, so it might be easier to update all aws-related deps